paymentsheet/src/main/java/com/stripe/android/paymentelement/confirmation/link/LinkConfirmationModule.kt

```diff
@@ -0,0 +1,31 @@
+package com.stripe.android.paymentelement.confirmation.link
+
+import com.stripe.android.link.LinkPaymentLauncher
+import com.stripe.android.link.account.LinkStore
+import com.stripe.android.link.injection.LinkAnalyticsComponent
+import com.stripe.android.paymentelement.confirmation.ConfirmationDefinition
+import dagger.Module
+import dagger.Provides
+import dagger.multibindings.IntoSet
+
+@Module(
+    subcomponents = [
+        LinkAnalyticsComponent::class,
+    ]
+)
+internal interface LinkConfirmationModule {
+    companion object {
```

Comment:
Nit, do you need the companion object here? Maybe just make `LinkConfirmationModule` a class?